### PR TITLE
Rc 1.4.3

### PR DESCRIPTION
**Pull request by semuadmin**
# PyGPSClient Pull Request Template

RELEASE v1.4.3

ENHANCEMENTS:

1. SPARTN MQTT configuration facilities enhanced to accommodate L-band as well as IP topics, including `/pp/frequencies/Lb`. As before, the utility will automatically upload the appropriate IP or L-band decryption keys and validity dates to the receiver (e.g. ZED-F9P) on receipt of an RXM-SPARTNKEY message (i.e. `/pp/ubx/0236/ip` or `/pp/ubx/0236/Lb` topic) from MQTT.
1. Note that a standard PointPerfect L-Band subscription does NOT provide access to the `/pp/ip/*` (SPARTN data) topics, the assumption being that the necessary correction data will arrive via RXM-PMP messages from the L-band receiver (e.g. NEO-D9S). Uncheck the IP topic checkbox before connecting to an L-band MQTT source.
1. Add support for MQTT SPARTN "jp" region.
1. Other minor improvements in usability.
1. Changes require pygnssutils>=1.0.15.

## Testing

Please test all changes, however trivial, against the supplied pytest suite `tests/test_*.py`. Please describe any test cases you have amended or added to this suite to maintain >= 99% code coverage.

## Checklist:

- [x] I agree to abide by the code of conduct (see [CODE_OF_CONDUCT.md](https://github.com/semuconsulting/pygpsclient/blob/master/CODE_OF_CONDUCT.md)).
- [x] My code follows the style guidelines of this project (see [CONTRIBUTING.MD](https://github.com/semuconsulting/pygpsclient/blob/master/CONTRIBUTING.md)).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] (*if appropriate*) I have added test cases to the `tests/test_*.py` unittest suite to maintain test coverage.
- [x] I have tested my code against the full `tests/test_*.py` unittest suite.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) my commits.
- [x] I understand and acknowledge that the code will be published under a BSD 3-Clause license.

### CODE REVIEW

File: src/pygpsclient/spartn_mqtt_frame.py [hunk 1]
```diff
@@ -257,6 +264,13 @@ def _do_layout(self):
         self._btn_connect.grid(column=0, row=9, padx=3, pady=2, sticky=W)
         self._btn_disconnect.grid(column=2, row=9, padx=3, pady=2, sticky=W)
 
+    def _attach_events(self):
```
Comment (semudev2) [inline src/pygpsclient/spartn_mqtt_frame.py:267]: Delete this function if it's not being used

File: src/pygpsclient/spartn_mqtt_frame.py [hunk 2]
```diff
@@ -117,6 +119,7 @@ def __init__(
 
         self._body()
         self._do_layout()
+        self._attach_events()
```
Comment (semudev2) [inline src/pygpsclient/spartn_mqtt_frame.py:122]: remove this if not being used

**Review by semudev2 — CHANGES_REQUESTED** [review]

File: src/pygpsclient/spartn_mqtt_frame.py (hunk 2 above)
Comment (semuadmin) [reply src/pygpsclient/spartn_mqtt_frame.py:122]: done

File: src/pygpsclient/spartn_mqtt_frame.py (hunk 1 above)
Comment (semuadmin) [reply src/pygpsclient/spartn_mqtt_frame.py:267]: done

**Review by semudev2 — APPROVED** [review]

**Pull request merged**